app/lib/dqt/params/update.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Dqt
+  module Params
+    class Update < TrnRequest
+    private
+
+      def build_params
+        {
+          "trn" => trainee.trn,
+          "husid" => trainee.hesa_id,
+          "initialTeacherTraining" => initial_teacher_training_params.merge(outcome_params),
+          "qualification" => qualification_params,
+        }
+      end
+
+      def outcome_params
+        { "outcome" => outcome }.compact
+      end
+
+      def outcome
+        return "Deferred" if trainee.deferred?
```

Comment:
If I understand this correctly, this means that:

1. If a user updates a field on a trainee after they're deferred, we tell DQT they are deferred (because their state is deferred on Register)
2. But when a user actually defers a trainee, we don't tell DQT they're deferred (because this is the update job which doesn't fire for deferrals)

See `ConfirmDeferralsController`:

```
def update
  if deferral_form.save!
    trainee.defer!
    flash[:success] = I18n.t("flash.trainee_deferred")
    redirect_to(trainee_path(trainee))
  end
end
```

`deferral_form.save!` <- DQT update job sent with outcome of `InTraining`
`trainee.defer!` <- Nothing sent to DQT

This is why I was a little confused - should this ticket actually include sending deferrals to DQT? Separately ticketed here: https://trello.com/c/m4cUsUcC/888-task-to-send-deferred-trainees-to-dqt-successfully